packages/flutter/test/material/mergeable_material_test.dart

```diff
@@ -55,7 +55,7 @@ BorderRadius getBorderRadius(WidgetTester tester, int index) {
   final List<Element> containers = tester.elementList(find.byType(Container))
                                    .toList();
 
-  final Container container = containers[index + 2].widget;
+  final Container container = containers[index].widget;
```

Comment:
Ah, I see. Please add a comment to that effect so that people examining this test the next time we change how Material works don't have to figure it out like you did. :-)